### ISSUE TITLE
Integrate API for getting the profile data in Client Side.

### ISSUE THREAD

**Issue by RishabhJain2018** [Cloud-CV/EvalAI#737]
When the user is directed to `My profile` after updating the profile, then the updated changes are not visible. The changes are visible only when the user again refreshes the page.

**Comment by spyshiv** [comment]
Can you elaborate it? 
If you are talking about getting user details then it is already done
endpoint: `auth/user/`
type: `GET`
required userkey also.

See the `frontend/src/js/directives/directives.js` line 69.

**Comment by RishabhJain2018** [comment]
@spyshiv The steps to regenerate it are:

1. Go to the update profile view once the PR #713 is merged.
2. Change the username or firstname or lastname.
3. Submit & then you will be redirected to `My Profile` view.

Now the data shown by the `My Profile` view is the old data. To get the new updated profile data , please refresh the `My Profile` view.

**Comment by spyshiv** [comment]
When someone get the update profile view then initially it should show the existing `username`, `first_name`, `last_name`. Then user can change it

**Comment by RishabhJain2018** [comment]
@spyshiv Yeah, I will update that.
Moreover, I opened this issue because even after updating the profile, the user can't see the chnages until he refreshes the page.

**Comment by RishabhJain2018** [comment]
@deshraj @spyshiv Can I work on this?

**Comment by deshraj** [comment]
Sure, assigning this to you. :)
